test/test_cpp_extensions_open_device_registration.py

```diff
@@ -599,9 +595,16 @@ def test_open_device_numpy_serialization(self):
         )
         buf_data_legacy_numpy = io.BytesIO(data_legacy_numpy)
 
+        class np_reconstruct:
+            __module__ = "numpy.core.multiarray"
+            __name__ = "_reconstruct"
+            __call__ = np.core.multiarray._reconstruct
+
         with safe_globals(
             [
-                np.core.multiarray._reconstruct,
+                np_reconstruct()
```

Comment:
This is a good example, but we also need a test to make sure the newly added API works right?

Reply:
Ho you already removed it :P